gm4_guidebook/beet.yaml

```diff
@@ -1,6 +1,6 @@
 id: gm4_guidebook
 name: Guidebook
-version: 2.2.X
+version: 3.0.X
```

Comment:
`text.gm4.guidebook.jump_to_section` needs to be added to the `translation_linter_ignores` list in this file. Also double check whether there are any translation keys that can be removed.